scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -232,16 +267,32 @@ trait TestScalaJSModule extends ScalaJSModule with TestModule {
     })
   }
 
+  @deprecated("To be removed. Use fastLinkJSTest instead", since = "0.10.4")
   def fastOptTest = T {
-    link(
-      ScalaJSWorkerApi.scalaJSWorker(),
-      scalaJSToolsClasspath(),
-      scalaJSTestDeps() ++ runClasspath(),
-      None,
+    linkJs(
+      worker = ScalaJSWorkerExternalModule.scalaJSWorker(),
+      toolsClasspath = scalaJSToolsClasspath(),
+      runClasspath = scalaJSTestDeps() ++ runClasspath(),
+      mainClass = None,
+      legacy = true,
```

Comment:
I maintain the old behavior in the legacy methods.

Reply:
If we use `forceOutJs` instead of `legacy`, it would be more obvious what exactly we enable here.

Reply:
If we use `forceOutJs` instead of `legacy`, it would be more obvious what exactly we enable here.